src/activity_item_handling.cpp

```diff
@@ -3290,6 +3348,46 @@ bool generic_multi_activity_handler( player_activity &act, Character &you, bool
         you.activity_vehicle_part_index = -1;
     }
     // scanned every location, tried every path.
+    if( !check_only && you.is_npc() ) {
+        if( src_sorted.empty() ) {
+            add_msg( m_neutral,
+                     _( "%1s failed to perform the %2$s activity because no suitable locations were found." ),
+                     you.disp_name(), activity_to_restore.c_str() );
+        } else if( reason.no_path ) {
+            add_msg( m_neutral,
+                     _( "%1$s failed to perform the %2$s activity because no path to a suitable location could be found." ),
+                     you.disp_name(), activity_to_restore.c_str() );
+        } else if( reason.skip_location_no_zone ) {
+            add_msg( m_neutral,
+                     _( "%1$s failed to perform the %2$s activity because no required zone was found." ),
+                     you.disp_name(), activity_to_restore.c_str() );
+        } else if( reason.skip_location_blocking ) {
+            add_msg( m_neutral,
+                     _( "%1$s failed to perform the %2$s activity because the target location is blocked or cannot be reached." ),
+                     you.disp_name(), activity_to_restore.c_str() );
+        } else if( reason.skip_location_no_skill ) {
+            add_msg( m_neutral, _( "%1$s failed to perform the %2$s activity because of unsufficient skills." ),
+                     you.disp_name(), activity_to_restore.c_str() );
+        } else if( reason.skip_location_unknown_activity ) {
+            add_msg( m_neutral,
+                     _( "%1$s failed to perform the %2$s activity because of the activity couldn't be found.  This is probably an error." ),
```

Comment:
```suggestion
                     _( "%1$s failed to perform the %2$s activity because the activity couldn't be found.  This is probably an error." ),
```